org.mwc.debrief.core/src/org/mwc/debrief/core/wizards/dynshapes/DynamicShapeBaseWizardPage.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * 
+ */
+package org.mwc.debrief.core.wizards.dynshapes;
+
+import java.util.regex.Pattern;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.FontData;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+
+/**
+ * @author Ayesha <ayesha.ma@gmail.com>
+ *
+ */
+public abstract class DynamicShapeBaseWizardPage extends WizardPage
+{
+  public static final String TIMINGS_PAGE = "Timings";
+  public static final String BOUNDS_PAGE = "Bounds";
+  public static final String STYLING_PAGE = "Styling";
+  private String pageName;
+  private static Font font;

```

Comment:
@ayeshare - I'm pretty sure (though I may be mistaken) that `font` and `regularFont` should not be made static.   In each `createBaseControl` we over-write the previous static instance.  Then when we call `dispose()` we may only be releasing the _newest_ font, not the overwritten one.

Fortunately, we can probably only have one dialog open at a time, so we haven't had any exceptions thrown, but I think we should correct it.

Reply:
done